openmdao/utils/coloring.py

```diff
@@ -516,8 +516,7 @@ def _get_bool_jac(prob, repeats=3, tol=1e-15, orders=5, setup=False, run_model=F
                 fullJ += np.abs(J)
         elapsed = time.time() - start_time
 
-    # normalize the full J by dividing by the max value
```

Comment:
is the comment still accurate?

Reply:
it's not accurate any more, and it was kind of unnecessary anyway since it was pretty obvious that the next line was dividing the matrix by the max.